android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/testing/CarJavaInterfaceChecker.java

```diff
@@ -73,4 +60,24 @@ public void onAttached(@NonNull MapboxCarMapSurface mapboxCarMapSurface) {
     mapboxCarMap.clearObservers();
   }
 
-}
+  private void gestures(MapboxCarMap mapboxCarMap) {
+    MapboxCarMapGestureHandler gestures = new MapboxCarMapGestureHandler() {
+      @Override
+      public boolean onScale(@NonNull MapboxCarMapSurface mapboxCarMapSurface, float focusX, float focusY, float scaleFactor) {
+        return true;
+      }
+
+      @Override
+      public boolean onFling(@NonNull MapboxCarMapSurface mapboxCarMapSurface, float velocityX, float velocityY) {
+        return true;
+      }
+
+      @Override
+      public boolean onScroll(@NonNull MapboxCarMapSurface mapboxCarMapSurface, @NonNull ScreenCoordinate visibleCenter, float distanceX, float distanceY) {
+        return true;
+      }
+    };
+    mapboxCarMap.setGestureDetector(gestures);
+    mapboxCarMap.setGestureDetector(new DefaultMapboxCarMapGestureHandler());
```

Comment:
do we allow multiple gesture detectors? I think the latter call will overwrite the first gestures detector?

Reply:
In this java interface checker, it would do nothing functionally. The second call will make the first one be forgotten

Reply:
Resolving this one